packages/app/src/components/home/componentRenderers/ComponentAccordion.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+  IconButton,
+  Theme,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import SettingsIcon from '@material-ui/icons/Settings';
+
+import { SettingsModal } from '@backstage/plugin-home';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  settingsIconButton: {
+    padding: theme.spacing(0, 1, 0, 0),
+  },
+}));
+
+export const ComponentAccordion = ({
```

Comment:
I guess open question as to whether you want to export these as part of the `home` plugin.  For now: probably doesn't matter where and it's fine here to iterate, but this one seems particularly generic / useful.

Reply:
That's a good point. For now I actually think it would make more sense in the `home` plugin. However, we want to emphasise that App Integrators can develop their own renderers when they're not satisfied with the way a Component renders. But perhaps that's more of a documentation thing. Fixed in f82b835